src/styles/emojiFont/index.js

```diff
@@ -0,0 +1,3 @@
+const emojiFont = 'Segoe UI Emoji, Noto Color Emoji';
```

Comment:
Lets move the emojiFont under `styles/fontFamily` folder and rename it to `emoji` only.